src/components/forms/CharacterCount/CharacterCount.test.tsx

```diff
@@ -335,11 +362,11 @@ describe('CharacterCount component', () => {
         />
       )
       expect(customMessage).toHaveBeenCalled()
-      expect(getByText('6 characters remain')).toBeInTheDocument()
+      expect(getAllByText('6 characters remain')[0]).toBeInTheDocument()
```

Comment:
**recommendation:** Assuming that this is because the srMessage is now also being retrieved in `getByText`. Is it worth checking for a .toHaveLength(2)?

Reply:
Yes. Good point. I will add that in.

Reply:
Thanks, added this in https://github.com/trussworks/react-uswds/pull/2550/commits/44089cfe6a5a801c3286de3c9783a87fee01754c